src/scripts/interface/dialogManager.ts

```
@@ -25,4 +27,22 @@ export class DialogManager {
       });
     });
   }
+
+  public static toggleDropdownDialog() {
+    if (this.dropdownDialog.open) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)